build/ci/set-up-ops-manager.sh

```diff
@@ -69,6 +71,15 @@ cat <<EOF > "${XDG_CONFIG_HOME}/mongocli.toml"
   private_api_key = "${MCLI_PRIVATE_API_KEY}"
   org_id = "${MCLI_ORG_ID}"
   project_id = "${MCLI_PROJECT_ID}"
+  agent_api_key = "${AGENT_API_KEY}"
+
+EOF
 
+popd
+cat <<EOF > automation_agent_settings.sh
+export BASE_URL=${MCLI_OPS_MANAGER_URL}
+export LC_AGENT_KEY=${AGENT_API_KEY}
+export LC_GROUP_ID=${MCLI_PROJECT_ID}
+export MCLI_SERVICE=${MCLI_SERVICE}
```

Comment:
[nb] technically this should be safe to always set to `ops-manager`, ins't it?

Reply:
yes